src/component/Edit/Editproperty.jsx

```diff
@@ -0,0 +1,239 @@
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
+import Selectoption from "../../Dashboard/Selectoption";
+import axios from "axios";
+
+const url = "http://localhost:3006/v1//property/:userId";
+
+const Editproperty = () => {
+ console.log(useParams())
```

Comment:
Remove this console.log since they are no longer useful

Reply:
worked on boss